extensions/wikia/Search/classes/Match/Article.php

```diff
@@ -3,57 +3,62 @@
  */
 namespace Wikia\Search\Match;
+
 use \Wikia\Search\Result as Result;
+
 /**
  * This class correlates a page ID to a MediaWiki article (via interface), and creates a result based on it.
+ *
  * @author relwell
  * @package Search
  * @subpackage Match
  */
-class Article extends AbstractMatch
-{
+class Article extends AbstractMatch {
 	/**
 	 * Creates a result instance.
+	 *
 	 * @see \Wikia\Search\Match\AbstractMatch::createResult()
 	 * @return Result
 	 */
 	public function createResult() {
-		/**@var $this->service Wikia\Search\MediaWikiService*/
+		/**@var $this ->service Wikia\Search\MediaWikiService */
 		$wikiId = $this->service->getWikiId();
 		$pageId = $this->service->getCanonicalPageIdFromPageId( $this->id );
 		$solrData = $this->getDataFromSolr( $wikiId, $pageId );
-		$fieldsArray = array(
-				'id'            => sprintf( '%s_%s', $wikiId, $pageId ),
-				'pageid'        => $pageId,
-				'wid'           => $wikiId,
-				'title'         => $this->service->getTitleStringFromPageId( $this->id ),
-				'url'           => $this->service->getUrlFromPageId( $this->id ) ,
-				'score'         => 'PTT',
-				'isArticleMatch'=> true,
-				'ns'            => $this->service->getNamespaceFromPageId( $this->id ),
-				'created'       => $this->service->getFirstRevisionTimestampForPageId( $this->id ),
-				'touched'       => $this->service->getLastRevisionTimestampForPageId( $this->id ),
-				'article_quality_i' => $solrData['article_quality_i']
-			);
+		$fieldsArray = [
+			'id' => sprintf( '%s_%s', $wikiId, $pageId ),
+			'pageid' => $pageId,
+			'wid' => $wikiId,
+			'title' => $this->service->getTitleStringFromPageId( $this->id ),
+			'url' => $this->service->getUrlFromPageId( $this->id ),
+			'score' => 'PTT',
+			'isArticleMatch' => true,
+			'ns' => $this->service->getNamespaceFromPageId( $this->id ),
+			'created' => $this->service->getFirstRevisionTimestampForPageId( $this->id ),
+			'touched' => $this->service->getLastRevisionTimestampForPageId( $this->id ),
+			'article_quality_i' => $solrData['article_quality_i']
+		];
 		$result = new Result( $fieldsArray );
 
-		$snippet =  $this->service->getSnippetForPageId( $this->id );
+		$snippet = $this->service->getSnippetForPageId( $this->id );
 
-		$result->setText($this->matchFoundText($snippet) );
+		$result->setText( $this->matchFoundText( $snippet ) );
 		if ( $this->hasRedirect() ) {
 			$result->setVar( 'redirectTitle', $this->service->getNonCanonicalTitleStringFromPageId( $this->id ) );
 			$result->setVar( 'redirectUrl', $this->service->getNonCanonicalUrlFromPageId( $this->id ) );
 		}
+
 		return $result;
 	}
 
 
-	protected function getDataFromSolr( $wikiId, $articleId){
+	protected function getDataFromSolr( $wikiId, $articleId ) {
 		$service = new \SolrDocumentService();
 		$service->setCrossWiki( false );
 		$service->setWikiId( $wikiId );
-		$service->setArticleId($articleId);
+		$service->setArticleId( $articleId );
 		$result = $service->getResult();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$result". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)